Validator/Validator.csproj

```diff
@@ -21,6 +21,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <Prefer32Bit>false</Prefer32Bit>
```

Comment:
@mgmccarthy Did you explicitly add this? I was unfamiliar with it so I did a bit of googling and I'm not sure if it is a change we want to make or not.